php/php.editor/src/org/netbeans/modules/php/editor/model/impl/Type.java

```diff
@@ -96,6 +96,7 @@ private Type() {
     public static final String PARENT = "parent"; //NOI18N
     public static final String STATIC = "static"; //NOI18N NETBEANS-4443 PHP 8.0
     public static final String NEVER = "never"; //NOI18N NETBEANS-5599 PHP 8.1
+    public static final String SPLIT_REGEX = "[()|&]"; // NOI18N
```

Comment:
Nitpick: `Pattern.compile()` should be used, for better performance.

Reply:
Yes, will fix it. Thanks!

Reply:
Add the following to the `CodeUtils`.
```java
    public static final Pattern WHITE_SPACES_PATTERN = Pattern.compile("\\s+"); // NOI18N
    public static final Pattern SPLIT_TYPES_PATTERN = Pattern.compile("[()|&]"); // NOI18N
```

Reply:
Cool, thanks.